starknet_py/tests/e2e/docs/account_creation/test_deploy_prefunded_account.py

```diff
@@ -29,6 +29,9 @@ async def test_deploy_prefunded_account(
 
     key_pair = KeyPair.from_private_key(private_key)
 
+    # You can also generate a key pair
+    key_pair_generated = KeyPair.generate()
+
```

Comment:
note: We could private separate section about different ways of key pair generation/reading, it would also cover `KeyPair.from_keystore()`. Can be addresses in separate PR as well.

Reply:
It sounds good and will be helpful for new users, let's create an issue for that

Reply:
Mentioned it in https://github.com/software-mansion/starknet.py/issues/1411.